atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableSplittingKeyValueService.java

```diff
@@ -89,6 +92,13 @@ public void createTable(String tableName, byte[] tableMetadata) {
 
     @Override
     public void createTables(Map<String, byte[]> tableNamesToTableMetadata) {
+        Map<KeyValueService, Map<String, byte[]>> splitTableNamesToTableMetadata = groupByDelegate(tableNamesToTableMetadata);
+        for (KeyValueService delegate : splitTableNamesToTableMetadata.keySet()) {
```

Comment:
This was already in there. Happy to change it to the entry set version, but didn't want to go around making things less clear for what looks like an irrelevant perf gain.